ago-dic-2019/Jesus Uriel Moncada Lara/Practica3/DinoClase.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+public class DinoClase {
+    public static void main(String args[]){
+          Scanner sc = new Scanner(System.in);
+          String arr[] = new String[5];  
+          
+                     String temporal;  
+             
+          
+             
+           System.out.print("Nombres de los dinosaurios\n : ");  
+           for(int i=0; i<arr.length; i++)  
+           {  
+               arr[i] = sc.next();  
+           }  
+             
+         
+           for(int i=0; i<arr.length; i++)  
+           {  
+               for(int j=i+1; j<arr.length; j++)  
+               {  
+                   if(arr[i].length() > arr[j].length())  
+                   {  
+                       temporal = arr[i];  
```

Comment:
Esto es como un selection sort, pero el tema es que cada vez que encuentras uno más pequeño haces el cambio, lo ideal sería que solamente lo marcaras y luego despues del for de la linea 20 haces un solo cambio